.github/workflows/tox-pytest.yml

```diff
@@ -19,10 +22,18 @@ jobs:
         with:
           fetch-depth: 2
 
+      - name: Set default GCP credentials
+        id: gcloud-auth
+        continue-on-error: true
+        uses: "google-github-actions/auth@v2"
+        with:
+          workload_identity_provider: "projects/345950277072/locations/global/workloadIdentityPools/gh-actions-pool/providers/gh-actions-provider"
+          service_account: "mozilla-dev-sa@catalyst-cooperative-mozilla.iam.gserviceaccount.com"
+
       - name: Set up conda environment for testing
         uses: mamba-org/setup-micromamba@v1
         with:
-          environment-file: environment.yml
+          environment-file: test_environment.yml
```

Comment:
Will the dependencies in this test environment also be kept up to date by things like dependabot? Or do we have to manually ensure that it tracks with `environment.yml`?

Reply:
I think it should be ok, since all of the important dependencies are defined in `pyproject.toml`. This whole thing is a bit weird, though, and maybe not the best approach? I found that all of our CI runs had started failing because the dependencies were too large for the github runner, and it turns out it's entirely because `pytorch` is intalling `cuda`, which uses > 6GB of disk space by itself. You can install a CPU only version of `pytorch`, which is much smaller, so I've forced `test_environment.yml` to use this version, but maybe there's a cleaner way to do this